README.md

```diff
@@ -42,39 +39,31 @@ Instructions for different operating systems can be found here:
 
 ## Roadmap
 
-The issues being planned and worked on are tracked using GitHub issues. To
-participate please visit:
+The issues being planned and worked on are tracked using GitHub issues.
+To give your input, visit:
 
 [MCUBoot GitHub Issues](https://github.com/mcu-tools/mcuboot/issues)
 
-~~Issues were previously tracked on [MCUboot JIRA](https://runtimeco.atlassian.net/projects/MCUB/summary)
-, but it is now deprecated.~~
-
 ## Browsing
 
 Information and documentation on the bootloader are stored within the source.
 
-~~It was previously also documented on confluence:
-[MCUBoot Confluence](https://runtimeco.atlassian.net/wiki/discover/all-updates)
-however, it is now deprecated and not currently maintained~~
-
-For more information in the source, here are some pointers:
+For more information on the source, see the following pointers:
 
 - [boot/bootutil](boot/bootutil): The core of the bootloader itself.
 - [boot/boot\_serial](boot/boot_serial): Support for serial upgrade within the bootloader itself.
 - [boot/zephyr](boot/zephyr): Port of the bootloader to Zephyr
 - [boot/mynewt](boot/mynewt): Mynewt bootloader app
 - [boot/nuttx](boot/nuttx): Bootloader application and port of MCUboot interfaces for NuttX.
 - [boot/espressif](boot/espressif): Bootloader application and MCUboot port for Espressif SoCs.
-- [imgtool](scripts/imgtool.py): A tool to securely sign firmware images for booting by mcuboot.
+- [imgtool](scripts/imgtool.py): A tool to securely sign firmware images for booting by MCUboot.
 - [sim](sim): A bootloader simulator for testing and regression
 
 ## Joining
 
-Developers welcome!
+Developers are welcome.
+Use the following links to join or see more about the project:
 
-* Our developer mailing list:
-  https://groups.io/g/mcuboot
-* Our Slack channel: https://mcuboot.slack.com/ <br />
-  Get your invite [here!](https://join.slack.com/t/mcuboot/shared_invite/MjE2NDcwMTQ2MTYyLTE1MDA4MTIzNTAtYzgyZTU0NjFkMg)
-* Our IRC channel: http://irc.freenode.net, #mcuboot
+* [Our developer mailing list](https://groups.io/g/MCUBoot)
+* [Our Slack channel](https://mcuboot.slack.com/)<br />
+  [Get your invite] (https://join.slack.com/t/mcuboot/shared_invite/MjE2NDcwMTQ2MTYyLTE1MDA4MTIzNTAtYzgyZTU0NjFkMg)
```

Comment:
There shouldn't be a space between the ']' and the '('.

Reply:
fixed